Update NetworkPolicy content

jboxman opened the pull request:
- NetworkPolicy is supported by other CNI plug-ins
- Drop excessive usage of code face for NetworkPolicy

jboxman commented:
@openshift/team-documentation, PTAL. Thanks!

bergerhoffer commented on modules/nw-networkpolicy-about.adoc:
```diff
@@ -6,33 +6,34 @@
 
 = About network policy
```
@jboxman Just wondering if this should be "About NetworkPolicy"? That seems to be how you're referring to it everywhere else.

jboxman replied:
Yes, thanks!

jboxman commented:
@vikram-redhat, should I close this and redo it against the new master once it's live?

vikram-redhat commented:
@jboxman up to you. Whatever you feel comfortable with.

jboxman commented:
Hi @zhaozhanqi, can you PTAL? I changed `NetworkPolicy` to NetworkPolicy in most places and updated the content in `modules/nw-networkpolicy-about.adoc` to clarify that other CNI plug-ins may support using NetworkPolicy too.

Thanks.

jboxman commented:
@danwinship @squeed 

I updated the language to reflect that NetworkPolicy is not exclusive to OpenShift SDN. Let me know if that makes sense.

(Preview: http://file.bos.redhat.com/~jboxman/20190613/networkpolicy-naming/networking/configuring-networkpolicy.html)

Thanks.

danwinship commented:
yes, looks right

jboxman commented:
/cherrypick enterprise-4.2

jboxman commented:
/cherrypick enterprise-4.1
